app/macos/Podfile.lock

```diff
@@ -100,4 +100,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce7dbe26c78bfc7ba46736094c1e2d25982870fa
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```

Comment:
we should be doing something about this coming up again and again ... 
```suggestion
COCOAPODS: 1.13.0
```

Reply:
Yes sure, I will revert it.

And yes need to do something about this type of configuration files changes. May we can add it git-ignore to discard the local changes while commit.

Reply:
@kumarpalsinh25  Agreed. This is somewhat annoying! :D